### PR TITLE
add H.265/HEVC 8K

### PR DESCRIPTION
**Pull request by paulhiggs**
This PR adds new classification schemes for VideoCodecCS:2021 and VideoConformancePointsCS:2021 which include 8K HEVC/H.265.
These CS's are already referenced by the A177r2 specification approved by TM last week.

### CODE REVIEW

**Comment by paulhiggs** [comment]
remove additional end tags - make formatting more 'user friendly'

File: metadata/cs/VideoCodecCS.xml [hunk 1]
```diff
@@ -1 +1,551 @@
-2020/VideoCodecCS.xml
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (c-alpha) [inline metadata/cs/VideoCodecCS.xml]: Can you make this file a symbolic link to your 2021 version, please?

File: metadata/cs/VideoConformancePointsCS.xml [hunk 2]
```diff
@@ -1 +1,146 @@
-2017/VideoConformancePointsCS.xml
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (c-alpha) [inline metadata/cs/VideoConformancePointsCS.xml]: Can you make this file a symbolic link to your 2021 version, please?

**Comment by c-alpha** [comment]
Hello Paul, many thanks for your swift action and this PR! Are you expecting to add further commits to this?

**Comment by paulhiggs** [comment]
No, that's it.
________________________________

**Review by c-alpha — CHANGES_REQUESTED** [review]

**Comment by paulhiggs** [comment]
Done. Also added another commit to update the DVB.zip bundle with the new CSs
________________________________

**Comment by c-alpha** [comment]
> Also added another commit to update the DVB.zip bundle with the new CSs.

Cool! I presume you used the `Makefile` to generate it?

**Comment by paulhiggs** [comment]
Um, no as I do not have 'zip' on my systems. Perhaps the ZIP file is something that can be created through CI/CD routines triggered on a merge.
________________________________

**Comment by c-alpha** [comment]
No worries; I can create the zip easily. There are a couple of further fixes I want to apply (bug 2872 & friends) before making the next release, so I'll wrap it into that.

Many thanks!

**Review by c-alpha — APPROVED** [review]

**Pull request merged**